temporal/t.rast.aggregate/t.rast.aggregate.html

```diff
@@ -193,6 +193,41 @@ <h4>Yearly aggregation</h4>
 yearly_avg_temp_2004|climate|2004-01-01 00:00:00|2005-01-01 00:00:00
 </pre></div>
 
+<h4>Weighted aggregation</h4>
+
+In this example, we create a STRDS of fictional temperature values for
+a weekly interval:
+
+<div class="code"><pre>
+MAPS="map_1 map_2 map_3 map_4 map_5 map_6 map_7 map_8 map_9 map_10"
+for map in ${MAPS} ; do
+    r.surf.random output=${map} min=278 max=298
+    echo ${map} >> map_list.txt
```

Comment:
```suggestion
for i in `seq 0 11` ; do
    r.mapcalc expression="map_${i} = ${i}"
    echo map_${i} >> map_list.txt
```

Reply:
This example makes it much easier to understand what the weight really does as values in input maps are known and constant integer values